popups.css

```diff
@@ -1,52 +1,47 @@
 .lsp_popup {
-    margin: 0.5rem 0.5rem 0 0.5rem;
+    margin: 0.5rem;
+    font-family: system;
 }
-.lsp_popup .highlight {
+.highlight {
     border-width: 0;
     border-radius: 0;
 }
-.lsp_popup p {
+.diagnostics {
     margin-bottom: 0.5rem;
-    padding: 0 0.5rem;
-    font-family: system;
-}
-.lsp_popup li {
-    font-family: system;
+    font-family: monospace;
```

Comment:
I'm not sure how to "reset" the font-family here. The value `monospace` doesn't actually do what it is supposed to do: https://github.com/sublimelsp/LSP/issues/1171#issuecomment-651800108

Reply:
You are referring to a deleted line?
Not sure what do you exactly mean since the code doesn't look like that anymore.

Reply:
I wrote `font-family: monospace;` here, but that doesn't actually work as you'd expect. It just so happens to work. But the behavior is unconfirmed on Windows.

The only special font-family is presumably `system`.